addon/templates/components/ilios-calendar.hbs

```diff
@@ -11,11 +11,12 @@
 </ul>
 <div class='ilios-calendar-calendar'>
   <h1 class="{{if calendarEventsPromise.isFulfilled 'loaded'}}">{{fa-icon 'spinner' spin=true}} {{loadingMessage}}</h1>
-  {{component (concat "ilios-calendar-" selectedView) 
+  {{component (concat "ilios-calendar-" selectedView)
     calendarEvents=calendarEventsPromise.content
     date=selectedDate
     selectEvent='selectEvent'
     changeDate='changeDate'
     changeView='changeView'
+    dueThisDay=dueThisDay
```

Comment:
dueThisDay needs to be set on ilios-calendar component

Like:
https://github.com/djvoa12/calendar/blob/display-fix/addon/components/ilios-calendar.js#L17